src/mcts/search.cc

```diff
@@ -282,13 +282,13 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) REQUIRES(counters_mutex_) {
     auto d = edge.GetD(default_d);
     float mu_uci = 0.0f;
     // Only the diff effect is inverted, so we only need to call if diff != 0.
-    if (params_.GetContemptPerspective() != ContemptPerspective::NONE) {
-      auto sign =
-          ((params_.GetContemptPerspective() == ContemptPerspective::STM) ||
-           ((params_.GetContemptPerspective() == ContemptPerspective::BLACK) ==
-            played_history_.IsBlackToMove()))
-              ? 1.0f
-              : -1.0f;
+    if (params_.GetWDLRescaleDiff() != 0.0f) {
```

Comment:
This check will cause `WDL_mu` to fail whenever contempt=0 is set, irrespective of the ContemptMode set.

Reply:
I looked at the comment above (`// Only the diff effect is inverted, so we only need to call if diff != 0.`), and a brief look through the code seemed to confirm it, obviously I was just confused. Should I remove the comment as well?

Reply:
Yes, comment predates the `mu_uci` population, so it is confusing (and wrong) by now.